src/components/calendarInput/index.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { DayPicker } from 'react-day-picker';
+import { addDays, addYears, endOfYear, format, startOfMonth } from 'date-fns';
+import { useSearchParams } from 'react-router-dom';
+
+import { ReactComponent as CalendarLogo } from 'assets/calendar.svg';
+
+import { setDate } from 'store/slices/inputsSlice';
+import { DayPickerWrapper } from './styled';
+import MainFiltersInput from '../common';
+import { useModal } from '../../hooks/useModal';
+
+const DATE_FORMAT_PATTERN = 'iii d MMM';
+
+export default function CalendarInput() {
+  const [isOpen, onOpenClick, onCloseClick] = useModal();
+
+  const { from, to } = useSelector(state => state.inputs.dates);
+  const [searchParams, setSearchParams] = useSearchParams();
+  const today = new Date();
+  const disabledDays = [
+    {
+      from: startOfMonth(today),
+      to: addDays(today, -1),
+    },
+  ];
```

Comment:
объявление переменных обычно происходит сразу перед использованием, перенеси к return
```suggestion
  const disabledDays = [
    {
      from: startOfMonth(today),
      to: addDays(today, -1),
    },
  ];
```

Reply:
ок, перенес